Backend/Models/Messages/TextMessage.cs

```diff
@@ -46,12 +47,39 @@ private TextMessage()
 
         public TextMessage(WebSocket socket, string rawString) : base(socket, rawString)
         {
-            _sender = _from;
-            _channel = _in;
+            _sender = _from != String.Empty ? _from : UserManager.GetUsernameBySocket(socket);
+            _channel = _in != String.Empty ? _in : "general-chat";
```

Comment:
Shouldn't we invalidate the message if these are missing?

Reply:
My original idea was that `FROM` doesn't need to be provided again after authentication, and it should be on the server to re-fill the username from the socket. That's how I documented it in the protocol 🤔

Reply:
Hmm, fair enough can we just set it to GetUsernameBySocket wihtout the check? How about channel then?

Reply:
Well there's going to be one public channel and then the dms, so i set it to the public channel "general-chat" by default for now.